packages/notifi-react-card/lib/components/subscription/NotifiSubscribeButton.tsx

```diff
@@ -106,7 +107,16 @@ export const NotifiSubscribeButton: React.FC<NotifiSubscribeButtonProps> = ({
 
   const renewTargetGroups = useCallback(async () => {
     if (isCanaryActive) {
-      return frontendClient.ensureTargetGroup(targetGroup);
+      let result = {} as TargetGroupFragmentFragment;
+      try {
+        result = await frontendClient.ensureTargetGroup(targetGroup);
+      } catch (e) {
+        // catch the case that the 'Default' discord already created
```

Comment:
Can you clarify this comment?

Do you mean the TargetGroup? rather than 'discord'